compiler/test/dotty/tools/dotc/interactive/CustomCompletionTests.scala

```diff
@@ -169,3 +169,21 @@ class CustomCompletionTests extends DottyTest:
 
       assert(offset == prefix.length)
       assert(labels.contains("scala.Function2"))
+
+   @Test def i12465_hkt(): Unit =
```

Comment:
Move these tests to `CompletionTest.scala` where they can be written in a much more concise way:
```
  @Test def i12465_hkt: Unit =
    code"""???.asInstanceOf[scala.collection.Seq].${m1}""".withSource
      .completion(m1, Set())

  @Test def i12465_hkt_alias: Unit =
    code"""???.asInstanceOf[Seq].${m1}""".withSource
      .completion(m1, Set())
```